beetsplug/discogs.py

```diff
@@ -341,18 +344,25 @@ def get_album_info(self, result):
                          day=None, label=label, mediums=len(set(mediums)),
                          artist_sort=None, releasegroup_id=master_id,
                          catalognum=catalogno, script=None, language=None,
-                         country=country, style=style,
+                         country=country, style=style, genre=genre,
                          albumstatus=None, media=media,
                          albumdisambig=None, artist_credit=None,
                          original_year=original_year, original_month=None,
                          original_day=None, data_source='Discogs',
-                         data_url=data_url)
+                         data_url=data_url,
+                         discogs_release_id=discogs_release_id,
+                         released_date=released_date)
 
-    def format_style(self, style):
-        if style is None:
-            self._log.debug('Style not Found')
+    def format(self, classification):
+        if classification is None:
+            self._log.debug('Classification not Found')
         else:
-            return self.config['separator'].as_str().join(sorted(style))
+            return self.config['separator'].as_str()\
+                .join(sorted(classification))
+
+    def extract_release_id(self, uri):
+        if uri:
+            return uri.split("/")[-1]
```

Comment:
It's good style to explicitly `return None` if the function's intent is to sometimes return `None`.